tasties_app/forms.py

```diff
@@ -16,3 +18,23 @@ class Meta:
             'username': forms.fields.TextInput(attrs={'class': 'form-control', 'placeholder': 'Username'}),
             'email': forms.fields.EmailInput(attrs={'class': 'form-control', 'placeholder': 'john@doe.com'}),
         }
+
+
+class CreateRecipeForm(ModelForm):
+    title = forms.CharField(max_length=100, widget=forms.TextInput(attrs={'class': 'form-control'}))
+    description = forms.CharField(max_length=1000, widget=forms.Textarea(attrs={'class': 'form-control', 'rows': 5}))
+    # ingredients = forms.CharField(max_length=1000, widget=forms.Textarea(attrs={'class': 'form-control'}))
```

Comment:
Why is this in a comment?

Reply:
Please see that this oversight was removed.